include/arch/arc/v2/aux_regs.h

```diff
@@ -110,6 +110,17 @@
 #define _ARC_V2_FPU_DPFP1H 0x303
 #define _ARC_V2_FPU_DPFP2L 0x304
 #define _ARC_V2_FPU_DPFP2H 0x305
+#define _ARC_V2_MPU_EN 0x409
+#if CONFIG_ARC_MPU_VER == 2
```

Comment:
no need for conditional compilations here, the values all are different and we don't have conditions for other HW features like e.g. FPU.